### DISCUSSION TITLE
Feature request- allow same output directory for multiple invocations

### DISCUSSION THREAD

**Discussion by ijhoskins** [ablab/spades#744] [Ideas]
I am using BayesHammer and noticed that the output directory is globbed on each invocation of spades.py. It would be handy for results for multiple samples to be written to the same output directory.

**Comment by asl** [comment]
Dear Ian,

What is the rationale for this? How this is supposed to work? Actually in SPAdes 3.14 it will refuse to run if the output directory is not empty to prevent spurious overwriting of the results.

**Comment by ijhoskins** [comment]
Hi @asl thanks for the reply. This is just a nice-to-have. My thought was that the results from multiple invocations could be dumped to a single output directory as opposed to multiple output directories. As it stands, if I set the same output directory for multiple invocations in serial, the results that remain are only those of the last invocation. If the plan is to throw an error if the output directory is not empty, maybe you could include a --force-overwrite flag to allow the user to make this choice at runtime.
